packages/workbox-core/INTERNAL/utils/LogHelper.mjs

```diff
@@ -43,13 +45,21 @@ export default class LogHelper {
   set logLevel(newLevel) {
     // TODO: Assert newLevel is a number
     if (typeof newLevel !== 'number') {
-      throw new Error('newLevel must be a number');
+      throw new WorkboxError('invalid-type', {
+        paramName: 'logLevel',
+        expectedType: 'number',
+        value: newLevel,
+      });
     }
 
     if (newLevel > ERROR_LOG_LEVEL ||
       newLevel < VERBOSE_LOG_LEVEL) {
-      // TODO: Throw workbox error + error code.
-      throw new Error(`Invalid new level`);
+      throw new WorkboxError('invalid-value', {
+        paramName: 'logLevel',
+        validValueDescription: `Please use a value from 'LOG_LEVEL', i.e ` +
```

Comment:
`'LOG_LEVEL'` -> `LOG_LEVELS`

(Drop the single-quotes around the getter name and add the `S`.)